docs/multicluster/user-guide.md

```diff
@@ -353,10 +359,19 @@ spec:
 
 ## Multi-cluster Gateway Configuration
 
-Multi-cluster Gateways are required to support multi-cluster Service access
-across member clusters. Each member cluster should have one Node served as its
-Multi-cluster Gateway. Multi-cluster Service traffic is routed among clusters
-through the tunnels between Gateways.
+Multi-cluster Gateways are responsible for establishing tunnels between clusters.
+Each member cluster should have one Node served as its Multi-cluster Gateway.
+Multi-cluster Service traffic is routed among clusters through the tunnels between
+Gateways.
+
+Below is a table about communication support with different combinations.
+
+| Pod-to-Pod connectivity provided by underlay | Gateway Enabled | MC EndpointTypes  | Cross-cluster Service/Pod communications |
+| -------------------------------------------- | --------------- | ----------------- | ---------------------------------------- |
+| No                                           | No              | None              | No                                       |
```

Comment:
If you think it is better, you can replace "None" with "N/A"

Reply:
Updated, thanks.